.github/workflows/nodejs-release.yml

```diff
@@ -1,56 +1,76 @@
-name: CI
+name: nodejs-release
 env:
   DEBUG: napi:*
-  APP_NAME: js-glaredb
-  MACOSX_DEPLOYMENT_TARGET: '10.13'
+  APP_NAME: "glaredb"
+  MACOSX_DEPLOYMENT_TARGET: "10.13"
+  PROTOC: "${{ github.workspace }}/deps/protoc/bin/protoc"
 permissions:
   contents: write
   id-token: write
-'on':
+on:
   push:
-    branches:
-      - main
-    tags-ignore:
-      - '**'
-    paths-ignore:
-      - '**/*.md'
-      - LICENSE
-      - '**/*.gitignore'
-      - .editorconfig
-      - docs/**
-  pull_request: null
+    tags:
+      - "*"
+  workflow_dispatch:
+defaults:
+  run:
+    working-directory: bindings/nodejs
+concurrency:
+  group: nodejs-release-${{ github.ref }}
+  cancel-in-progress: true
+
 jobs:
   build:
     strategy:
-      fail-fast: false
+      fail-fast: true
       matrix:
         settings:
-          - host: macos-latest
+          - host: macos-12-xl
             target: x86_64-apple-darwin
             build: |
               yarn build
               strip -x *.node
-          - host: windows-latest
+          - host: windows-latest-8-cores
             build: yarn build
             target: x86_64-pc-windows-msvc
-          - host: ubuntu-latest
+          - host: ubuntu-2004-8-cores
             target: x86_64-unknown-linux-gnu
             docker: ghcr.io/napi-rs/napi-rs/nodejs-rust:lts-debian
             build: |-
-              set -e &&
+              set -eux &&
+              curl --proto '=https' --tlsv1.2 -sSf https://just.systems/install.sh | bash -s -- --to . &&
+              ./just protoc && 
+              export PROTOC="$PWD/deps/protoc/bin/protoc" &&
+              cd bindings/nodejs &&
               yarn build --target x86_64-unknown-linux-gnu &&
               strip *.node
+          - host: macos-12-xl
+            target: aarch64-apple-darwin
+            build: |
+              rustup target add aarch64-apple-darwin
+              sudo rm -Rf /Library/Developer/CommandLineTools/SDKs/*;
+              export CC=$(xcrun -f clang);
+              export CXX=$(xcrun -f clang++);
+              SYSROOT=$(xcrun --sdk macosx --show-sdk-path);
+              export CFLAGS="-isysroot $SYSROOT -isystem $SYSROOT";
+              yarn build --target aarch64-apple-darwin
+              strip -x *.node
```

Comment:
Was this copied from somewhere? If so, lets throw that link in so someone can reference it if necessary.

Reply:
this is generated from napi.